FlymFork/src/main/res/values/strings.xml

```diff
@@ -392,7 +392,7 @@
     <string name="text">Text</string>
     <string name="background1">Background</string>
     <string name="textColor">Text color</string>
-    <string name="textColorRead">Text color read</string>
+    <string name="textColorRead">Color of articles which have been read</string>
```

Comment:
Too long for prerefence title. It will be cut. Try to shorten it

Reply:
I think Your initial idea is better, so let;'s leave _"Read"_ and _"Read articles"_.